ethcore/build.rs

```diff
@@ -17,7 +17,7 @@
 extern crate ethcore_ipc_codegen;
 
 fn main() {
-	ethcore_ipc_codegen::derive_binary("src/types/mod.rs.in").unwrap();
+	ethcore_ipc_codegen::derive_binary_cond("src/types/mod.rs.in", cfg!(feature="ipc")).unwrap();
```

Comment:
i don't 100% understand the use of adding the flag parameter here. having a function take a bool parameter saying whether it should run is pretty strange. I think it's also already covered with the `cfg_attr` guards that have been added everywhere.